tests/fuzz/s2n_kyber_r2_fuzz_test.c

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
You'll need to update https://github.com/awslabs/s2n/blob/master/tests/fuzz/runFuzzTest.sh#L60 to exclude Kyber fuzz tests when in FIPS mode.

Reply:
updated.